kubernetes/keda.libsonnet

```diff
@@ -0,0 +1,51 @@
+local metric_definition(type, queue='', value='', activationThreshold='0', server) =
```

Comment:
please be consistent with the function name casing. We're using camel case pretty much everywhere throughout this library. Don't randomly switch naming to snake case.

Reply:
Done